src/plugins/intel_cpu/src/ngraph_transformations/snippets_disable_subgraph_transforms.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include <memory>
+#include <ngraph/pass/graph_rewrite.hpp>
+#include <snippets/op/subgraph.hpp>
+#include <vector>
+
+namespace ov {
+namespace intel_cpu {
+
+class SnippetsDisableSubgraphTransforms: public ngraph::pass::MatcherPass {
```

Comment:
```suggestion
class SnippetsDisableSubgraphTransforms: public pass::MatcherPass {
```
MatcherPass is in `ov` namespace.